rest-fights/pom.xml

```diff
@@ -150,8 +150,8 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>io.smallrye.reactive</groupId>
-      <artifactId>smallrye-reactive-messaging-in-memory</artifactId>
```

Comment:
The in-memory connector is needed. The unit tests use the in-memory connector while the integration tests use a "real" Kafka consumer.

Removing this causes the tests to no longer compile.

Reply:
Sorry about that, I didn't run other tests after my rebase.